Project.toml

```diff
@@ -11,6 +11,7 @@ CubedSphere = "7445602f-e544-4518-8976-18f8e8ae6cdb"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+Enzyme = "7da242da-08ed-463a-9acd-ee780be4f1d9"
```

Comment:
so now Enzyme is both a normal and a weak dependendcy?

Reply:
Well I'm not sure if it was accidental or on purpose. But adding it there implies that's a core dependency of Oceananigans. I'll have a look.

Reply:
x-ref #3452